Felienne/hedy
# [FEATURE] Adds download option to multiple accounts creation

**Description**
In this PR we add a small but useful feature. We enable teacher to download the user login credentials at the moment of multiple accounts creation. This way teachers can easily print these and give them physically to their students or keep them to not forget the passwords. We add a radio button option, when selecting "yes" a .csv file is dynamically created on the front-end being automatically downloaded with the name `accounts.csv`.

**Fixes**
This PR fixes #2843.

**How to test**
Make sure you are a teacher and navigate to the `multiple accounts` page. Verify that there is a new radio button to select if you want to download the login credentials after creation. Try this and a .csv file should automatically be downloaded with the corresponding data.
